documentation/LezEditor.md

```diff
@@ -0,0 +1,51 @@
+The LEZ editor allow users to draw and edit low emission zones perimeters, that
+can then be downloaded and give as [input to the SmartGovLezModel](https://github.com/smartgov-liris/SmartGovLezModelUFD/blob/master/documentation/inputs/lez.md).
+
+![LEZ](lez.png)
+
+# Input Data
+
+![LEZ Data](lez-data.png)
+
+Establishments can be imported, in order to visualize them editing the LEZ.
+This is optional and has no impact on the LEZ output file.
+
+Establishments should be **json output files** produced by the
+[SmartGovLezModel](https://github.com/smartgov-liris/SmartGovLezModelUFD/blob/master/documentation/outputs/Standard-Outputs.md#establishments),
+what should node be confused with establishments *input* data of the
+SmartGovLezModel.
```

Comment:
Does it have to be SmartGovLezModelUFD (in place of SmartGovLezModel) everywhere in this documentation !?

Reply:
The links have been fixed (they point to SmartGovLezModelUFD), but not the words. I'll fix that.